be/src/exec/pipeline/pipeline_driver.cpp

```diff
@@ -470,6 +470,16 @@ void PipelineDriver::_update_statistics(size_t total_chunks_moved, size_t total_
         query_ctx()->incr_cur_scan_rows_num(source->get_last_scan_rows_num());
         query_ctx()->incr_cur_scan_bytes(source->get_last_scan_rows_num());
     }
+
+    // Update cpu cost of this query
+    int64_t runtime_ns = driver_acct().get_last_time_spent();
+    int64_t source_operator_last_cpu_time_ns = 0;
+    if (SourceOperator* source = source_operator()) {
```

Comment:
A pipeline driver is guaranteed to contain at least 2 drivers source and sink. Maybe we needn't judge whether source is nullptr.

Reply:
Good advice. I have removed the check. The only concern is test cases did not construct the sink operator. But since the UT could pass, I could remove it anyway.